src/components/Utils.jsx

```diff
@@ -0,0 +1,12 @@
+export const firstArticle = {
```

Comment:
Better name this file utils.js as it's not a React component
![image](https://user-images.githubusercontent.com/96409563/230439338-4a3e4525-bd6e-48ff-b999-0f62efcd47b2.png)